### ISSUE TITLE
unable to round trip floating point numbers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jbeder/yaml-cpp#272]
```
The code used to control the precision of floating point numbers is too 
restrictive, the limit should be defined more like this

(2 + std::numeric_limits<float>::digits * 3010/10000)

see

http://www2.open-std.org/JTC1/SC22/WG21/docs/papers/2005/n1822.pdf

not sure if the default should be changed, for consistency I would
```

Original issue reported on code.google.com by `Kevin.J....@gmail.com` on 30 Jan 2015 at 1:57

Attachments:
- [yaml-cpp.precision.diff](https://storage.googleapis.com/google-code-attachments/yaml-cpp/issue-272/comment-0/yaml-cpp.precision.diff)

**Comment by KevinJW** [comment]
Looks like #470 might also address this.

Kevin

**Comment by jbeder** [comment]
Fixed by #649.
